src/main/groovy/edu/oregonstate/mist/trailsapi/resources/TrailResources.groovy

```diff
@@ -101,4 +101,15 @@ public class TrailsResource extends Resource {
 			zipCode, smallDrop,largeDrop, woodRide, skinny, largeJump,smallJump, gap)
 		ok(trailResult(trails)).build()
 	}
+
+	@GET
+    @Path ('/{id: \\d+}')
```

Comment:
The indenting is a bit off with this line. Just curious, are you using intellij Idea for this API?

Reply:
I've been using Atom as I prefer the UI compared to Idea. I started this learning project with some bad whitespace settings that have since been corrected, so it should not be an issue in the future